dev/com.ibm.websphere.appserver.features/visibility/public/mpJwt-1.2/io.openliberty.mpJwt-1.2.feature

```diff
@@ -8,17 +8,17 @@ IBM-API-Package: \
   org.eclipse.microprofile.auth; type="stable"
 Subsystem-Name: MicroProfile JSON Web Token 1.2
 -features=com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:=4.0, \
-  com.ibm.websphere.appserver.appSecurity-2.0; ibm.tolerates:=3.0, \
+  com.ibm.websphere.appserver.appSecurity-3.0; ibm.tolerates:=4.0, \
```

Comment:
4.0 is for Jakarta 9.  Since you aren't tolerating servlet 5.0 I assume you don't need to tolerate appSecurity 4.0

Reply:
also it should only be servlet 4.0.

Reply:
Right, Jakarta EE 8 features only for MicroProfile 4.0. No additional tolerates.